pom.xml

```diff
@@ -3,7 +3,7 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>4.61</version>
+    <version>4.62</version>
```

Comment:
Please, for next time, avoid introducing unrelated changes, as it makes the review work harder and increases the likelihood of merge conflicts.

Reply:
Thanks for the note! Currently rebased to latest upstream and this bump is no longer included with PR.